swig/nvector/fnvector_parallel_mod.i

```diff
@@ -41,14 +41,14 @@
 }
 
 %typemap(in, noblock=1) MPI_Comm {
-%#ifdef SUNDIALS_MPI_ENABLED
+%#if SUNDIALS_MPI_ENABLED
```

Comment:
Should this be removed like in the manyvector files?

Reply:
Yes